mcc/ttl/ttl_test.go

```diff
@@ -113,3 +113,22 @@ func TestUpsertChangeList(t *testing.T) {
 		}
 	}
 }
+
+var fstest = []string{
+	"",
+	"a,b",
+	"a,,a,,c,,",
+}
+
+func TestFilterSet(t *testing.T) {
+	for _, s := range fstest {
+		t.Run(s, func(t *testing.T) {
+			var f Filter
+			_ = f.Set(s)
+			j := strings.Join(f, ",")
+			if s != j {
+				t.Error(j, " doesn't match")
```

Comment:
It is a subtest so, if it fails, it will show a text saying:
```
s Failed --
j doesn't match
```
Which I think already informs me that s and j should match and that they don't.